python/pyspark/mllib/linalg.py

```diff
@@ -897,6 +914,33 @@ def __init__(self, numRows, numCols, colPtrs, rowIndices, values,
             raise ValueError("Expected rowIndices of length %d, got %d."
                              % (self.rowIndices.size, self.values.size))
 
+    def __str__(self):
```

Comment:
Could you add a doc test for `__str__`, it's easy to see the result in doc test.